app/controllers/api/subcollections/alert_definitions.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  module Subcollections
+    module AlertDefinitions
+      def alert_definitions_query_resource(object)
+        return {} unless object.respond_to?(:miq_alerts)
+        object.miq_alerts
```

Comment:
Sorry I was confused with policies (I keep thinking of the two together). This looks good, we only need to get the miq alerts belonging to a miq alert profile